pyproject.toml

```
@@ -1,11 +1,79 @@
 [build-system]
 requires = [
-    "setuptools>=42",
+    "setuptools @ git+https://github.com/abravalheri/setuptools@support-pyproject",
     "wheel",
     "setuptools_scm[toml]>=3.4.3"
 ]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "plumbum"
+description = "Plumbum: shell combinators library"
+readme = "README.rst"
+author = { name="Tomer Filiba", email="tomerfiliba@gmail.com" }
+license = { file="LICENSE" }
+requires-python = ">=3.6"
+dynamic = ["version"]
+requires = [
```

PEP 621 seems to use `dependencies` instead of `requires`.

```suggestion
dependencies = [
```